netci.groovy

```diff
@@ -2515,6 +2575,46 @@ combinedScenarios.each { scenario ->
                                         }
                                     }
                                 }
+                                else if ( architecture == 'arm' && ( os == 'Ubuntu' || os == 'Ubuntu16.04' || os == 'Tizen')) {
+                                    // Cross build for ubuntu-arm, ubuntu16.04-arm and tizen-armel
+                                    // Define the Windows Tests and Corefx build job names
+                                    def WindowTestsName = projectFolder + '/' +
+                                                          Utilities.getFullJobName(project,
+                                                                                   getJobName(lowerConfiguration,
+                                                                                              'x64' ,
+                                                                                              'windows_nt',
+                                                                                              'default',
+                                                                                              true),
+                                                                                   false)
+                                    def corefxFolder = Utilities.getFolderName('dotnet/corefx') + '/' +
+                                                       Utilities.getFolderName(branch)
+
+                                    // Copy the Windows test binaries and the Corefx build binaries
+                                    copyArtifacts(WindowTestsName) {
+                                        excludePatterns('**/testResults.xml', '**/*.ni.dll')
+                                        buildSelector {
+                                            latestSuccessful(true)
+                                        }
+                                    }
+
+                                    // Defaults for Ubuntu
+                                    def arm_abi='arm'
+                                    def corefx_os='ubuntu14.04'
+                                    if (os == 'Ubuntu16.04') {
+                                        arm_abi="arm"
+                                        corefx_os='ubuntu16.04'
+                                    }
+                                    else if (os == 'Tizen') {
+                                        arm_abi='armel'
+                                        corefx_os='tizen'
+                                    }
+                                    copyArtifacts("${corefxFolder}/${corefx_os}_${arm_abi}_cross_${lowerConfiguration}") {
```

Comment:
Oops, I almost missed it.
Thank you for the feedback and I've updated this as you suggested :)